_startup/chiche.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ¡Chiche!
+mission: Recrutez des gens, pas des CV
+domain: chiche.beta.gouv.fr
+repository: 
+status: investigation
```

Comment:
Rien ou presque... La fiche produit est encore sujette à itération. Mais
pas de souci avec "construction", seulement un regret qu'on ne voie pas le
produit basculer d'une étape à la suivante (on aurait du attendre moins
longtemps pour le déclarer sur beta). Je vais amender, merci pour le
feedback.

2016-04-21 17:56 GMT+02:00 Matti Schneider <notifications@github.com>:

> In _startup/chiche.md
> <https://github.com/sgmap/beta.gouv.fr/pull/150#discussion_r60607221>:
>
> > @@ -0,0 +1,13 @@
> > +---
> > +title: ¡Chiche!
> > +mission: Recrutez des gens, pas des CV
> > +domain: chiche.beta.gouv.fr
> > +repository:
> > +status: investigation
>
> Si on a la fiche produit, le porteur administratif, l'équipe et le budget,
> dans la taxonomie actuelle c'est construction. Que manque-t-il
> aujourd'hui ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sgmap/beta.gouv.fr/pull/150/files/9c110bfd3a9d314b27bf6c01a783b89a94688f63#r60607221>
>